GEOSldas_App/ldas_setup

```diff
@@ -1112,6 +1124,14 @@ class LDASsetup:
                     sp.call(shlex.split(cmd))
                 for line in fileinput.input(tmprcfile,inplace=True):
                     print (line.rstrip().replace('GEOSldas_expid',self.rqdExeInp['EXP_ID']))
+                if (self.ladas_coupling == 1) :
+                    for line in fileinput.input(tmprcfile,inplace=True):
+                        print (line.rstrip().replace("#                'catch_progn_incr'","   'catch_progn_incr'"))
+                    for line in fileinput.input(tmprcfile,inplace=True):
+                        print (line.rstrip().replace('catch_progn_incr.ref_time:    000000,','catch_progn_incr.ref_time:    013000,'))
+                    for line in fileinput.input(tmprcfile,inplace=True):
+                        print (line.rstrip().replace("'SMAP_L4_SM_gph'","# 'SMAP_L4_SM_gph'"))
+
```

Comment:
It might make more sense to add a new template of GEOSldas_HIST.rc that is specific to the LADAS, rather than edit GEOSldas_HIST.rc in this way

Reply:
can we add a new template  for central and  one for atmens  in  GEOSldas_App/ , so that they will be installed along with GEOSldas_HIST.rc  in  install-SLES15/etc/  by adas build.